docs/rst/user_manual/configuration.rst

```diff
@@ -14,16 +14,22 @@ and :term:`Participants <Participant>` configurations.
 Topic Filtering
 ===============
 
-.. note::
-
-    The |ddsrouter| topic discovery module is a work in progress.
-    Thus, the functionality regarding Topic filtering is still in its early stages.
-
-The |ddsrouter| requires a list of allowed :term:`Topics<Topic>`.
-The data transmitted under these topics will be the data relayed by the |ddsrouter|.
-The YAML configuration file must contain an ``allowlist`` tag which is a vector (``[]``).
-This vector elements contains the Topics that will be forwarded (subsequently the |ddsrouter| creates a
-:term:`Writers<DataWriter>` and :term:`Readers<DataReader>` for each topic for each Participant).
+|ddsrouter| allows filtering of DDS :term:`Topics<Topic>`, that is, it allows defining which DDS Topics are going to be
+relayed by the application.
+This way, it is possible to define a set of rules in |ddsrouter| to filter those data that the user do not want to be
+forwarded. It is not mandatory to define a rule set in the configuration file.
+In this case, a DDS Router will forward all the data published under the topics that it automatically discovers within
+the DDS network to which it connects.
+
+To define these data filtering rules based on the Topics to which they belong, three lists are available:
+
+* Builtin topics list (``builtin-topics``)
+* Allowed topics list (``allowlist``)
+* Block topics list (``blocklist``)
```

Comment:
Maybe better explain `builtin-topics` the last one, and let's try to use this ordering everywhere else.